net/scripts/solana-user-authorized_keys.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# Contains the public keys for users that should automatically be granted access
+# to ALL testnets and datacenter nodes.
+#
+# To add an entry into this list:
+# 1. Run: ssh-keygen -t ecdsa -N '' -f ~/.ssh/id-solana-testnet
+# 2. Add an entry to SOLANA_USERS with your username
+# 3. Add an entry to SOLANA_PUBKEYS with the contents of ~/.ssh/id-solana-testnet.pub
+
+SOLANA_USERS=()
+SOLANA_PUBKEYS=()
+
+SOLANA_USERS+=('mvines')
+SOLANA_PUBKEYS+=('ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBFBNwLw0i+rI312gWshojFlNw9NV7WfaKeeUsYADqOvM2o4yrO2pPw+sgW8W+/rPpVyH7zU9WVRgTME8NgFV1Vc=')
+
+SOLANA_USERS+=('sathish')
+SOLANA_PUBKEYS+=('ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBGqZAwAZeBl0buOMz4FpUYrtpwk1L5aGKlbd7lI8dpbSx5WVRPWCVKhWzsGMtDUIfmozdzJouk1LPyihghTDgsE=')
+
+SOLANA_USERS+=('carl')
+SOLANA_PUBKEYS+=('ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBOk4jgcX/VWSk3j//wXeIynSQjsOt+AjYXM/XZUMa7R1Q8lfIJGK/qHLBP86CMXdpyEKJ5i37QLYOL+0VuRy0CI=')
+
+SOLANA_USERS+=('jack')
+SOLANA_PUBKEYS+=('ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBEB6YLY4oCfm0e1qPswbzryw0hQEMiVDcUxOwT4bdBbui/ysKGQlVY8bO6vET1Te8EYHz5W4RuPfETbcHmw6dr4=')
+
+SOLANA_USERS+=('trent')
+SOLANA_PUBKEYS+=('ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIEZC/APgZTM1Y/EfNnCHr+BQN+SN4KWfpyGkwMg+nXdC trent@fry')
+
+SOLANA_USERS+=('tristan')
+SOLANA_PUBKEYS+=('ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC0+3//c4pSjMzysddEQFiRWwow8iyPuwkAJ2CDSuAFHzZ22XBdzghwNyJz97suaiefOc5hAgPWTeM8p0NpHsPItgRlwqljxDrQDKEW2K79j3/rnS8h0OJWAnoJ5bX8ft1h+C3mtUJtOWN67wGSpIOIxo1j/zHJHKJZSMmSxEtd4FzdUsMdezEeF1rDak2z4UlpEX1Y7rir8I2D4uvpLRBBCyW49NX/dAv+cpNoJD6eb1Kt85bfB3VTO5Qv8gsMnAruM+OQ2GnYUE1OgHR3miPSyQ7S0NvwKTe/k8L24dlNzy6Hw+jlQiqE/2O5llBZc07I878WyqMMV+Wxk9nQFyWT tristan@TristanSolanaMacBook.local')
```

Comment:
RSA keys 😱

Reply:
If I'm reading [this](https://tools.ietf.org/id/draft-ietf-curdle-ssh-kex-sha2-10.html#rfc.section.3.28) correctly, the 2048bit rsa keys should be acceptable, but I do have to change mine.

Reply:
IIRC 2048-bit RSA is more of a *minimum*, 3072 *recommended*, 4096+ *paranoid*.

@mvines is probably complaining about them being a mile long, though :wink:

Reply:
If I'm reading [this](https://tools.ietf.org/id/draft-ietf-curdle-ssh-kex-sha2-10.html#rfc.section.3.28) right, the 2048bit keys should be ok, but I do have to change mine